queue/deliverer.go

```diff
@@ -358,7 +363,7 @@ func wrapResource(originatingUser string, msg syslog.Message) logging.Resource {
 	if m := msg.Timestamp(); m != nil {
 		lm.LogTime = m.Format(logging.TimeFormat)
 	}
-	parseProcID(msg, &lm)
+	parseProcID(msg, &lm) // Commenting out the code which overwrites applicationInstanceId
```

Comment:
remove this line yes

Reply:
modified the function to remove the else check